apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -153,11 +184,14 @@ export const PoolTable = ({ pools }: PoolTableProps) => {
           description:
             "Header for the View Candidates column of the Pools table",
         }),
-        accessor: (pool) =>
+        accessor: (d) => {
+          return d.id;
+        },
```

Comment:
If we are retuning just the ID this might cause some confusion on sorting. I think it may be best to just disable it for this column with `disableSortBy`. Possible search as well, it is causing some issues where I get unexpected results.

Reply:
disableSortBy makes the header look not good 😬 


![image](https://user-images.githubusercontent.com/40485260/217311252-52460791-1774-4829-916c-a8d4c2a8ca73.png)

Reply:
Okay, not in scope of this issue so I'll create a ticket for that.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/5642